### ISSUE TITLE
[question] should `skip_merge` also include `Merge pull request...` messages

### ISSUE THREAD

**Issue by hamza-shezad** [mergeability/mergeable#721]
when `skip_merge` is `true`, should commits with messages as following be skipped?

```
Merge pull request #111 from <repo>`
```

**Comment by rm98** [comment]
Docs https://github.com/mergeability/mergeable/blob/590a40fa9ab9a5f3adbc11273687f043beb91aec/docs/validators/commit.rst#L10

and implementation
https://github.com/mergeability/mergeable/blob/590a40fa9ab9a5f3adbc11273687f043beb91aec/lib/validators/commit.js#L69
```js
      orderedCommits = orderedCommits.filter(commit => !commit.message.includes('Merge branch'))
```

diverge. `Merge` vs. `Merge branch`.

=> Fix the implementation?

**Comment by hamza-shezad** [comment]
yes. i think removing `branch` will be enough. do you want me to open a PR?

**Comment by rm98** [comment]
Yes, I would suggest that you open a PR and ping the maintainers!
